provision/testdata/cloud_tar/cluster-network-03-config.yaml

```diff
@@ -6,7 +6,9 @@ metadata:
 spec:
   disableMultiNetwork: true
   clusterNetwork:
-  - cidr: "14.3.0.1/16"
+  - cidr: [
+    "14.3.0.1/16"
+]
```

Comment:
This might be a problem - I don't think OpenShift cluster network operator supports a list:
https://github.com/openshift/cluster-network-operator

I think we will not be able to support multiple subnets for the OpenShift case, meaning, we throw an error (at least for now) if multiple subnets are configured in the case of the OpenShift flavor. Secondly, the above would always be just a single subnet, not a list.

@abhis2112 any thoughts on the above?

Reply:
Correct Sumit. The way the network operator supports multiple  pod networks is like below:

ClusterNetwork: []configv1.ClusterNetworkEntry{
		{
			CIDR:       "10.0.0.0/22",
			HostPrefix: 24,
		},
		{
			CIDR:       "10.2.0.0/22",
			HostPrefix: 23,
		},
	},


The CIDR can't be a list. It's a single subnet string. Any right now we are supporting a single subnet and also in the case of multus is on a different note and not configured like aforementioned.